common/culture/cultures/wc_orcish.txt

```diff
@@ -122,14 +172,18 @@ stormreaver = {
 	color = rgb { 133 181 177 }
 	
 	parents = { shadowmoon }
-
 	created = 589.4.12
 	
 	ethos = ethos_spiritual
 	heritage = heritage_orcish
 	language = language_orcish
 	martial_custom = martial_custom_equal
 	traditions = {
+		tradition_seafaring
```

Comment:
From the [Stormreaver WoWpedia entry](https://wowpedia.fandom.com/wiki/Stormreaver_clan#The_Second_War):

> During the course of the Second War, the Stormreavers held domain over [Stormwind City](https://wowpedia.fandom.com/wiki/Stormwind_City) and the island of [Balor](https://wowpedia.fandom.com/wiki/Balor).[[1]](https://wowpedia.fandom.com/wiki/Stormreaver_clan#cite_note-WC2_Manual-1) After the Horde's successful conquest of [Khaz Modan](https://wowpedia.fandom.com/wiki/Khaz_Modan), Orgrim prepared to build a fleet to launch a surprise attack in the heart of [humanity](https://wowpedia.fandom.com/wiki/Human)'s territories. Many orcs had reservations about crossing the sea, but the Stormreavers urged their allies that it was the best course of action and that the voyage would be safe. Orgrim welcomed the assistance but remained wary of Gul'dan's motivations.[[4]](https://wowpedia.fandom.com/wiki/Stormreaver_clan#cite_note-4) The Stormreavers also captured [giant sea turtles](https://wowpedia.fandom.com/wiki/Submarine#Goblin_giant_turtles) from the [South Seas](https://wowpedia.fandom.com/wiki/South_Seas), to be used as a submersible orcish craft after being pacified by potent spells of control.[[5]](https://wowpedia.fandom.com/wiki/Stormreaver_clan#cite_note-5) According to [Aegwynn](https://wowpedia.fandom.com/wiki/Aegwynn), Gul'dan and the Stormreavers were rumored to be amassing a great navy in order to find the Tomb of Sargeras.[[6]](https://wowpedia.fandom.com/wiki/Stormreaver_clan#cite_note-6)

Otherwise, my suggested replacements are Religious Patronage or Sacred Hunts.

Reply:
> Otherwise, my suggested replacements are Religious Patronage or Sacred Hunts.

Nah, `tradition_seafaring` sounds good.